api/openapi.ts

```diff
@@ -752,141 +1025,2376 @@ export type external = Record<string, never>;
 
 export interface operations {
   /**
-   * Create an app
-   * @description Create an app.
+   * Get billing account information.
+   * @description Get billing account information for a given team.
    */
-  "apps-create": {
+  "admin-billingAccount-get": {
+    parameters: {
+      readonly path: {
+        /** @description The team's ID */
+        teamId: string;
+      };
+    };
+    responses: {
+      /** @description Successful response */
+      200: {
```

Comment:
should we add potential error codes?

Reply:
this looks like a good example on how to handle error responses https://stackoverflow.com/questions/60318034/is-there-a-way-to-get-the-error-response-object-when-using-generated-webclient-a